dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -216,9 +218,9 @@ public void testComponent_staticLinkageCheckResult() throws IOException, Parsing
 
       Nodes staticLinkageCheckMessage = document.query("//p[@class='jar-linkage-report']");
       Assert.assertEquals(1, staticLinkageCheckMessage.size());
-      Truth.assertThat(staticLinkageCheckMessage.get(0).getValue())
-          .contains("2 linkage errors in 2 classes");
-
+      Truth.assertThat(toList(staticLinkageCheckMessage))
```

Comment:
This feels like a less specific test than what it replaces.

Reply:
Updated with new function `trimAndCollapseWhiteSpaces`.